tools/data_dictionary_builder.php

```diff
@@ -169,7 +169,7 @@
             //for HTML_QuickForm versions of standard HTML Form Elements...
         default:
             //continue; // jump straight to validity for debugging
-            if (preg_match("/^Examiner/", $bits[1])) {
+            if (isset($bits[1]) && preg_match("/^Examiner/", $bits[1])) {
```

Comment:
This might be cleaner using null coalesce on the bits instead of isset. ie. `preg_match("...", $bits[1] ?? "");`